### ISSUE TITLE
Add support for ebess/advanced-nova-media-library

### ISSUE THREAD

**Issue by bendadaniel** [Muetze42/nova-single-resource#9]
Add support for ebess/advanced-nova-media-library please.

Thank you

**Comment by Muetze42** [comment]
Yes, you have to take a look. Spatie saves it to the ID of a line... Of course it will be a bit more "tricky" to think of something useful...
